CaseStudy.md

```diff
@@ -0,0 +1,166 @@
+# Case Study – “Accelerating ML Model Development and Deployment on the edge with the Machine Learning Model Factory”
+
+
+## Table of Contents
```

Comment:
None of the images have resolution clarity? Unable to read the text in the images

Reply:
Added png format images with better resolution & clarity. Please check once